portfolio/portfolio_test.go

```diff
@@ -346,20 +350,9 @@ func TestStartPortfolioWatcher(t *testing.T) {
 
 	assert.NoError(t, b.AddAddress(testXRPAddress, PersonalAddress, currency.XRP, 0.02))
 
-	ctx, cancel := context.WithTimeout(t.Context(), time.Second)
-	defer cancel()
-
-	doneCh := make(chan error)
-	go func() {
-		doneCh <- b.StartPortfolioWatcher(ctx, 0) // 0 will set the ticker to 10 minutes
-	}()
-
-	select {
-	case err := <-doneCh:
-		assert.ErrorIs(t, err, context.DeadlineExceeded, "StartPortfolioWatcher should return DeadlineExceeded error")
-	case <-time.After(2 * time.Second):
-		assert.Fail(t, "StartPortfolioWatcher should have deadlined")
-	}
+	ctx, cancel := context.WithTimeout(t.Context(), 0)
+	cancel()
```

Comment:
🐒 Not sure why we're using a Timeout
```suggestion
	ctx, cancel := context.WithCancel(t.Context())
	cancel()
```